cypress/integration/conditions/numeric_conditions.cy.js

```diff
@@ -0,0 +1,286 @@
+        .find('tbody > tr')
+        .should('have.length', expectedRows.length)
+
+    expectedRows.forEach((val) => {
+        getLineListTable().find('tbody').contains('td', val)
+    })
+}
+
+const assertTooltipContainsEntries = (entries) => {
+    entries.forEach((entry) =>
+        cy.getWithDataTest('{tooltip-content}').contains(entry)
+    )
+}
+
+describe('number conditions', () => {
+    beforeEach(() => {
+        cy.visit('/', EXTENDED_TIMEOUT)
+        setUpTable()
+    })
+    it('equal to', () => {
+        addConditions([{ conditionName: 'equal to (=)', value: '1232' }])
+        assertTableMatchesExpectedRows(['1 232'])
+
+        cy.getWithDataTest('{layout-chip}')
+            .contains(`${dimensionName}: 1 condition`)
+            .trigger('mouseover')
+
+        assertTooltipContainsEntries([
+            'Program stage: Birth',
+            'Equal to (=): 1 232',
+        ])
+    })
+
+    it('greater than', () => {
+        addConditions([{ conditionName: 'greater than (>)', value: '3980' }])
+
+        assertTableMatchesExpectedRows(['3 988', '3 999'])
+
+        cy.getWithDataTest('{layout-chip}')
+            .contains(`${dimensionName}: 1 condition`)
+            .trigger('mouseover')
+
+        assertTooltipContainsEntries([
+            'Program stage: Birth',
+            'Greater than (>): 3 980',
+        ])
+    })
+
+    it('greater than or equal to', () => {
+        addConditions([
+            { conditionName: 'greater than or equal to', value: '3980' },
+        ])
+        assertTableMatchesExpectedRows(['3 988', '3 980', '3 999'])
+
+        cy.getWithDataTest('{layout-chip}')
+            .contains(`${dimensionName}: 1 condition`)
+            .trigger('mouseover')
+
+        assertTooltipContainsEntries([
+            'Program stage: Birth',
+            'Greater than or equal to (≥): 3 980',
+        ])
+    })
+
+    it('less than', () => {
+        addConditions([{ conditionName: 'less than (<)', value: '2615' }])
+        assertTableMatchesExpectedRows(['1 232', '2 561'])
+
+        cy.getWithDataTest('{layout-chip}')
+            .contains(`${dimensionName}: 1 condition`)
+            .trigger('mouseover')
+
+        assertTooltipContainsEntries([
+            'Program stage: Birth',
+            'Less than (<): 2 615',
+        ])
+    })
+
+    it('less than or equal to', () => {
+        addConditions([
+            { conditionName: 'less than or equal to', value: '2615' },
+        ])
+        assertTableMatchesExpectedRows(['1 232', '2 561', '2 615'])
+
+        cy.getWithDataTest('{layout-chip}')
+            .contains(`${dimensionName}: 1 condition`)
+            .trigger('mouseover')
+
+        assertTooltipContainsEntries([
+            'Program stage: Birth',
+            'Less than or equal to (≤): 2 615',
+        ])
+    })
+
+    it('not equal to', () => {
+        addConditions([{ conditionName: 'not equal to', value: '1232' }])
+
+        assertTableMatchesExpectedRows([
```

Comment:
Could we maybe find test data with less expected rows? 20+ rows feels tricky to debug if one happens to be missing for some reason in the future... I suggest something in the range of 3-7 rows should be enough to test it properly but few enough to be able to debug easily.